src/components/RightPane/CoursePane/CourseRenderPane.js

```diff
@@ -91,43 +96,45 @@ const flattenSOCObject = (SOCObject) => {
     }, []);
 };
 
-const RecruitmentBanner = ({ className }) => {
+const RecruitmentBanner = (classes) => {
     // Idk how else to force a function component to update
     const [, updateState] = React.useState();
     const forceUpdate = React.useCallback(() => updateState({}), []);
 
+    // Display recruitment banner if more than 11 weeks (in ms) has passed since last dismissal
     let displayRecruitmentBanner =
-        !Boolean(window.localStorage.getItem('dismissedRecruitmentBanner')) &&
+        Date.now() - window.localStorage.getItem('recruitmentDismissalTime') > 6652800000 &&
```

Comment:
nit: the dismissal time would be easier to edit as an expression instead of the calculated result. e.g. `11 * 7 * 24 * 3600 * 1000`. This is kinda opinionated though, so you can change this if you agree or leave it as-is.

Reply:
^ This would make it easy to change the number of weeks later on